homework/add-even/addEven.hpp

```diff
@@ -2,7 +2,16 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    
+    auto sum =0;
+
+    for(auto& element : numbers){
+        auto division_rest = element%2;
+        if (division_rest == 0){
+        sum += element;
+        }
+    }
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..